tests/functions.py

```diff
@@ -0,0 +1,72 @@
+from os import listdir, walk
+from os.path import isdir, exists, join, abspath, basename, dirname
+from random import random
+import datalad.api as api
+
+
+def recurse(directory, odds):
```

Comment:
You should add a docstring to explain what the function does and what `odds` is.